contracts/seed/SeedFactory.sol

```diff
@@ -105,18 +112,22 @@ contract SeedFactory is CloneFactory, Ownable {
                     _tokens.length == 2 &&
                     _softAndHardCap.length == 2 &&
                     _startTimeAndEndTime.length == 2 &&
-                    _defaultClassParameters.length == 3,
+                    _defaultClassParameters.length == 3 &&
+                    _projectAddresses.length == 2,
                 "SeedFactory: Error 102"
```

Comment:
In the long run, could each of this boolean be split up into its own require?
Because, if I get 102 back, then it's hard to know which of these conditions was actually causing the issue

Reply:
Yes we can do that